eng/common/darc-init.sh

```diff
@@ -68,7 +68,7 @@ function InstallDarcCli {
     fi
   fi
 
-  local arcadeServicesSource="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json"
+  local arcadeServicesSource="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json"
```

Comment:
Changes to eng/common must not be made in the VMR. You probably want to make this change in src/arcade/eng/common/darc-init.sh (which probably is already done). That's the location that flows to repos in and outside of the VMR.